pkgs/top-level/all-packages.nix

```diff
@@ -10986,6 +10986,8 @@ in
 
   dhex = callPackage ../applications/editors/dhex { };
 
+  dnnl = callPackage ../development/libraries/dnnl { };
```

Comment:
Can you, perhaps, also include an alias to mkldnn? I think most people will not be aware of this name switch to dnnl and may be confused.